modules/acquia_cms_site_studio/composer.json

```
@@ -4,8 +4,8 @@
     "license": "GPL-2.0-or-later",
     "type": "drupal-module",
     "require": {
-        "acquia/cohesion": "~7.4.0",
-        "acquia/cohesion-theme": "~7.4.0",
+        "acquia/cohesion": "~7.5.0",
```

@rajeshreeputra Let's use both versions (unless previous is breaking). 
```suggestion
        "acquia/cohesion": "~7.4.0 || ~7.5.0",
```
and same for site_studio_config_management module.

---

7.5 supports GIN theme dark and light mode, whereas 7.4 does not. So both versions can not be supported as the same time.